package/src/components/Attachment/Giphy.tsx

```diff
@@ -85,8 +85,8 @@ const styles = StyleSheet.create({
     width: '60%',
   },
   selectionContainer: {
-    borderRadius: 16,
     borderBottomRightRadius: 0,
+    borderRadius: 16,
```

Comment:
as far as I know, the order should be

```
borderRadius: 16,
borderBottomRightRadius: 0,
```

So that the override works

Reply:
Did you try to run it? It seemingly works just fine when I run it on my end. Our linting rules also enforce this ordering of the keys keeping them in alphabetical order.

<img width="287" alt="image" src="https://user-images.githubusercontent.com/1548276/164888883-4b991305-901f-417c-a42d-731f8c7ed5a0.png">

Reply:
No i did not try and run it on RN, but tried now on CSS as it was easier. For overriding in CSS, the ordering is important. Tried the following,

```
<!DOCTYPE html>
<html>
<head>
<style> 
#example1 {
  border: 2px solid red;
  padding: 10px;
  border-bottom-right-radius: 0px;
  border-radius: 25px;
}

#example2 {
  border: 2px solid red;
  padding: 10px;
  border-radius: 25px;
  border-bottom-right-radius: 0px;
}
</style>
</head>
<body>

<h2>border-bottom-right-radius: 0px; border-radius: 25px</h2>
<div id="example1">
  <p>doesnt work</p>
</div>

<h2>border-radius: 25px; border-bottom-right-radius: 0px;</h2>
<div id="example2">
  <p>works</p>
</div>

</body>
</html>
```

Can we disable the lint rule here and keep the ordering like this so that it may not break in the future?

Reply:
Yeah its a bit wierd for me as well that RN doesn't respect the ordering. But it's done deliberately within RN I think. Original ordering isn't taken into account, but all these properties are set explicitly on view manager - https://github.com/facebook/react-native/blob/c6daadcab4387de85fb0070502a6e898d7f473a9/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java#L117
So view manager isn't aware of the original order of these styles.

And if they change this, its going to be quite a massive breaking change everywhere. So I suggest lets not worry about it :)